community-membership.md

```diff
@@ -0,0 +1,213 @@
+# Community membership
+
+**Note:** This document is in progress
+
+This doc outlines the various responsibilities of contributor roles in
+Operator Framework. The Operator Framework project is subdivided into subprojects.
+Responsibilities for most roles are scoped to these subprojects.
+
+| Role | Responsibilities | Requirements | Defined by |
+| -----| ---------------- | ------------ | -------|
+| member | active contributor in the community | sponsored by 2 reviewers, multiple contributions to the project | Operator Framework GitHub org member. |
+| reviewer | review contributions from other members | history of review and authorship in a subproject | [OWNERS] file reviewer entry. |
+| approver | approve accepting contributions | highly experienced and active reviewer + contributor to a subproject | [OWNERS] file approver entry|
+| subproject owner | set direction and priorities for a subproject | demonstrated responsibility and excellent technical judgment for the subproject | [sigs.yaml] subproject [OWNERS] file *owners* entry |
+
+## New contributors
+
+New contributors should be welcomed to the community by existing members,
+helped with PR workflow, and directed to relevant documentation and
+communication channels.
```

Comment:
do we have a "getting started for contributors" doc?  If so, we should link it here (if not, lemme add a TODO).

Reply:
No, will add a TODO.

Reply:
Here's what we currently have. This doc should be mentioned somewhere in these docs
https://github.com/operator-framework/operator-sdk/blob/master/CONTRIBUTING.MD
https://sdk.operatorframework.io/docs/contribution-guidelines/